test/python/docker/compat/common.py

```diff
@@ -1,5 +1,5 @@
 """
-Fixtures and Helpers for unittests.
+Fixtures and Helpers for unit tests.
```

Comment:
Python expert wanted here: I believe in Pythonland it is idiomatic to write it as one word, because that's the name of the module. This needs review by someone proficient in Pythonspeak.

Reply:
I reverted the change as I think you have a valid point about spelling in Pythonland.